### PR TITLE
Adding cephfs IOs to upgrade suites

### PR DESCRIPTION
**Pull request by yogesh-mane**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>
Adding logs for 2 suites as fix is just adding cephfs IOs

Logs for suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container.yaml - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-THYA6J 
tier-1_upgrade_cephadm_without_rgw.yaml  - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-C6QCNN/Upgrade_along_with_IOs_0.log

### CODE REVIEW

File: suites/pacific/upgrades/tier-1_upgrade_4x-to-5x-baremetal-without-dashboard.yaml [hunk 1]
```diff
@@ -161,6 +161,15 @@ tests:
         io-total: 100M
     desc: Perform export during read/write,resizing,flattening,lock operations
 
+- test:
+    abort-on-fail: false
+    config:
+        timeout: 30
```
Comment (AmarnatReddy) [inline suites/pacific/upgrades/tier-1_upgrade_4x-to-5x-baremetal-without-dashboard.yaml:167]: Can we increase it to 600 seconds

**Comment by hkadam134** [comment]
@yogesh-mane - You have increased the timeout value in only 1 suite file, can you modify it in all the suites where the changes have been made?

**Comment by yogesh-mane** [comment]
@manasagowri , could you please review this PR

**Review by hkadam134 — APPROVED** [review]
Looks Good to me

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**